src/demo/models/guide.js

```diff
@@ -4,53 +4,86 @@ import {AppMode, Modes} from '../constants';
 const guides = [
   {
     id: 'fishvtrash-training-init',
-    text: "Let's clean up the ocean!",
+    text: "Garbage dumped in the ocean and rivers affects water health and marine life.  In this activity you will program an artificial intelligence (AI) to identify trash.  Let's clean up the ocean!",
     when: {appMode: AppMode.FishVTrash, currentMode: Modes.Training},
-    style: 'TopLeft',
+    style: 'Center',
     arrow: 'none'
   },
   {
     id: 'fishvtrash-training-init2',
-    text: 'Use these two buttons to train A.I.',
+    text: "Let's meet A.I.",
     when: {appMode: AppMode.FishVTrash, currentMode: Modes.Training},
-    style: 'BottomMiddleButtons'
+    style: 'TopRight',
   },
   {
-    id: 'fishvtrash-training-done',
-    text: "Great work!  You can continue when you're ready.",
+    id: 'fishvtrash-training-init3',
+    text: 'A.I. does not know if an object is a fish or trash, but it can process different images and identify patterns.',
+    when: {appMode: AppMode.FishVTrash, currentMode: Modes.Training},
+    style: 'BottomRightCenter',
+    arrow: 'none'
+  },
+  {
+    id: 'fishvtrash-training-init4',
+    text: "To program A.I., use the buttons to label an image as either 'fish' or 'not fish'. This will train A.I. to do it on it's own. Let's get started!",
```

Comment:
nit: suggest using `'` to surround text so that we can have `"` around the words.

Reply:
Also `it's` should be `its`.